cli/ballerina-cli/src/test/resources/test-resources/command-outputs/unix/build-bal-project-with-build-tool-not-found.txt

```diff
@@ -1,6 +1,3 @@
 
 Executing Build Tools
 ERROR [Ballerina.toml:(6:1,9:26)] Build tool 'grpc' cannot be resolved
-
```

Comment:
Isn't the `build tool execution contains errors` part getting printed?

Reply:
It is printed properly.